pom.xml

```diff
@@ -159,6 +164,140 @@
           </testResource>
         </testResources>
         <plugins>
+        <!--
+		    This plugin helps enforce that excluded dependencies stay excluded 
+			and don't sneak back in through another dependency (a global ban). Here are 
+			a few common situations this plugin helps with:
+			
+			1) multiple implementations of a dependency - you only want to use one implementation
+			   (ex: commons-logging vs jcl-over-slf4j, javax.mail vs geronimo-javamail_1.4_spec)
+			2) similar maven artifacts are produced for the same project - you only need to include one 
+			   (ex: cglib vs cglib-nodep)
+			3) an artifact's group id or artifact id has been renamed - maven considers these different
+			   artifacts and you only want to include one
+			   (ex: opensymphony:quartz vs org.opensymphony.quartz:quartz vs org.quartz-scheduler:quartz)
+			4) an artifact was added to the container and no longer is needed
+			   (ex: jaxb in java 6) This plugin is also enforcing maven and jdk versions.
+	      -->
+		  <plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-enforcer-plugin</artifactId>
+			<version>1.0-beta-1</version>
+			<executions>
+			  <execution>
+				<id>enforce-banned-dependencies</id>
+				<goals>
+				  <goal>enforce</goal>
+				</goals>
+				<configuration>
+				  <rules>
+					<bannedDependencies>
+						<excludes>
+							<!-- renamed (decrecated) groupids -->
+							<exclude>wss4j</exclude>
+							<exclude>acegisecurity</exclude>
+							<exclude>xstream</exclude>
+							<exclude>uk.ltd.getahead</exclude>
+							<exclude>quartz</exclude>
+							<exclude>groovy</exclude>
+									
+							<!-- already including cglib-nodep -->
+							<exclude>cglib:cglib</exclude>
+							<!-- already including the jdk16 version 
+							<exclude>org.bouncycastle:bcprov-jdk14</exclude>
+							 already including the jdk16 version
+							<exclude>org.bouncycastle:bcprov-jdk15</exclude> -->
+
```

Comment:
Those exclude lines can be removed, unless you think we should upgrade bcprov-jdk to version 1.46?.  Currently bcprov-jdk15-1.45 is a dependency so if I left those excludes in the build fails.   1.46 is the version that is being used in the Rice 2.6.   

We have the following dependency:

      <dependency>
        <groupId>org.bouncycastle</groupId>
        <artifactId>bcprov-jdk15</artifactId>
        <version>${bcprov.version}</version>     <--- 1.45 for Rice 1.0, 1.46 for Rice 2.6
      </dependency>